ts/Dashboard/Component/Component.ts

```diff
@@ -489,29 +492,45 @@ abstract class Component<TEventObject extends Component.EventTypes = Component.E
    * @return {this}
    * The component for chaining
    */
-    public update(newOptions: Partial<Component.ComponentOptions>): this {
+    public update(newOptions: Partial<Component.ComponentOptions>, redraw = true): this {
         // Update options
         this.options = merge(this.options, newOptions);
         fireEvent(this, 'update', {
             options: newOptions
         });
 
+        if (redraw) {
+            this.redraw();
+        }
+
         return this;
     }
 
     public setTitle(titleOptions: Component.TextOptionsType): void {
+        const old = this.titleElement;
```

Comment:
I think that the `old` is too general, can we replace it with something more meaningful? F.i. - `prevTitle` or `currentTitle`